api/src/main/java/org/openmrs/Program.java

```diff
@@ -122,14 +122,41 @@ public String toString() {
 	// Property Access
 	// ******************
 	
+	/**
+	 * @deprecated As of 2.0.1
+	 * 
+	 * replaced by {@link #getName())}
+	 */
+	 @deprecated
```

Comment:
this is the culprit. There is no `@deprecated` annotation. This is used only inside javadocs. Like you did above. What you are looking for is `@Deprecated`. Notice capitalization of the D.